### ISSUE TITLE
What is the correct representation of unassigned device?

### ISSUE THREAD

**Issue by petertuharsky** [pluginsGLPI/datainjection#380]
Hallo.

I'm trying to import our devices to GLPI from CSV. Some devices are not assigned to any user. Please, how to correctly address such situation in the CSV, so that plugin dosen't assign the device to some bogus user? I did try to find an advice in the documentation, but failed.

**Comment by stonebuzz** [comment]
Hi @petertuharsky 

Have you tried not to put anything in the "User" column?

**Comment by petertuharsky** [comment]
Yes, sometimes it takes the first manually created user or so. We use Active Directory user base.
For now, we already did our migration, but it might be relevant for someone else.
